ragna/source_storages/_lancedb.py

```diff
@@ -43,31 +43,54 @@ def __init__(self) -> None:
         import pyarrow as pa
 
         self._db = lancedb.connect(ragna.local_root() / "lancedb")
-        # Create schema at runtime!
-        self._schema = pa.schema(
-            [
-                pa.field("id", pa.string()),
-                pa.field("document_id", pa.string()),
-                pa.field("page_numbers", pa.string()),
-                pa.field("text", pa.string()),
-                pa.field(
-                    self._VECTOR_COLUMN_NAME,
-                    pa.list_(pa.float32(), self._embedding_dimensions),
-                ),
-                pa.field("num_tokens", pa.int32()),
-            ]
-        )
+
+        fields = [
+            pa.field("id", pa.string()),
+            pa.field("document_id", pa.string()),
+            pa.field("document_name", pa.string()),
+            pa.field("page_numbers", pa.string()),
+            pa.field("text", pa.string()),
+            pa.field(
+                self._VECTOR_COLUMN_NAME,
+                pa.list_(pa.float32(), self._embedding_dimensions),
+            ),
+            pa.field("num_tokens", pa.int32()),
+        ]
+
+        self._schema = pa.schema(fields)
```

Comment:
Nit: make it concise

```suggestion
        self._schema = pa.schema(
            [
                pa.field("id", pa.string()),
                pa.field("document_id", pa.string()),
                pa.field("document_name", pa.string()),
                pa.field("page_numbers", pa.string()),
                pa.field("text", pa.string()),
                pa.field(
                    self._VECTOR_COLUMN_NAME,
                    pa.list_(pa.float32(), self._embedding_dimensions),
                ),
                pa.field("num_tokens", pa.int32()),
            ]
        )
```